### PR TITLE
Docs/raptor

### PR DESCRIPTION
**Pull request by andre-merzky**

### CODE REVIEW

**Review by mturilli — APPROVED** [review]
LGTM!

File: docs/source/apidoc.rst [hunk 1]
```diff
@@ -62,6 +62,13 @@ Task
 .. autoclass:: radical.pilot.Task
    :members:
 
+Raptor
+------
+.. autoclass:: radical.pilot.raptor.Master
+   :members:
+.. autoclass:: radical.pilot.raptor.Worker
+   :members:
+
```
Comment (eirrgang) [inline docs/source/apidoc.rst:71]: Note that `Pilot.submit_tasks()` and `Pilot.submit_raptors()` is not documented (as of RP 1.34, at least)

```suggestion

A `radical.pilot.Task` managing a `radical.pilot.raptor.Master` instance is created using
``mode=rp.RAPTOR_MASTER``, or through :py:func:`~radical.pilot.Pilot.submit_raptors()`.
The object returned to the client is a `Task` subclass with additional features.

.. autoclass:: radical.pilot.raptor_tasks.Raptor
    :members:
```

File: docs/source/apidoc.rst [hunk 2]
```diff
@@ -62,6 +62,20 @@ Task
...
+A `radical.pilot.Task` managing a `radical.pilot.raptor.Master` instance is created using
+:py:attr:`radical.pilot.TaskDescription.mode` 
+``rp.RAPTOR_MASTER``, or through :py:func:`~radical.pilot.Pilot.submit_raptors()`.
+The object returned to the client is a `Task` subclass with additional features.
+
+.. autoclass:: radical.pilot.raptor_tasks.Raptor
+    :members:
```
Comment (eirrgang) [inline docs/source/apidoc.rst:78]: Oops.
```suggestion
    :members:

```

**Review by eirrgang — COMMENTED** [review]
There are a few more Sphinx tweaks necessary. I'll have an update in a few minutes...

File: docs/source/apidoc.rst (hunk 2 above)
Comment (eirrgang) [reply docs/source/apidoc.rst:78]: Ref #3004

**Pull request merged**